hera_pspec/tests/test_pspecdata.py

```diff
@@ -219,17 +273,93 @@ def test_get_G(self):
                 # the other.
                 G_swapped = self.ds.get_G(key2, key1, taper=taper)
                 G_diff_norm = np.linalg.norm(G - G_swapped)
-                self.assertLessEqual(G_diff_norm, matrix_scale*10**-10)
-
-
+                self.assertLessEqual(G_diff_norm, matrix_scale*1e-10)
                 min_diagonal = np.min(np.diagonal(G))
+                
                 # Test that all elements of G are positive up to numerical noise
                 # with the threshold set to 10 orders of magnitude down from
                 # the smallest value on the diagonal
                 for i in range(Nfreq):
                     for j in range(Nfreq):
-                        self.assertGreaterEqual(G[i,j], -min_diagonal*10**-10)
+                        self.assertGreaterEqual(G[i,j], -min_diagonal*1e-10)
             
+    def test_parseval(self):
```

Comment:
I'm actually no longer sure that this is the right test. Let's talk about it in person.